static/docs/commands-reference/commit.md

```diff
@@ -186,11 +186,9 @@ deps:
   path: data/features
   md5: b91b22bfd8d9e5af13e8f48523e80250
 outs:
-- cache: true
```

Comment:
let's keep everywhere where is was specified before set `cache` and `metric`, you can remove other default values